blocks/authAPI/views.py

```diff
@@ -34,6 +36,25 @@ def activate_user(request, uid, token):
                    })
 
 
+def pwd_reset(request, uid, token):
+    """
+    Used to reset password,
+    sends POST request to /api/auth/users/reset_password_confirm/ route
+    internally to reset user password.
+    Link to this route is sent via email to user for verification
+    """
+
+    protocol = 'https://' if request.is_secure() else 'http://'
+    web_url = protocol + request.get_host() + '/api/auth/users/reset_password_confirm/'  # Djoser endpoint
```

Comment:
```suggestion
    web_url = settings.POST_ACTIVATE_REDIRECT_URL + 'api/auth/users/reset_password_confirm/'  # Djoser endpoint
```
replace with POST_ACTIVATE_REDIRECT_URL